vita3k/modules/SceLibKernel/SceKernelForMono.cpp

```diff
@@ -49,9 +49,8 @@ EXPORT(int, sceKernelWaitExceptionCBForMono) {
 EXPORT(int, sceKernelWaitExceptionForMono) {
     TRACY_FUNC(sceKernelWaitExceptionForMono);
     STUBBED("Inifinite wait");
```

Comment:
I think this stub can be modified now

Reply:
It's still infinite wait, just by other way.

Reply:
Ah indeed you're right.